.github/workflows/ci-17.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Test
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+
+    name: Build and Test (17)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Also instead of duplicating the test information for each one of them, can we use github action matrix?

Something like
```
jobs:
  test:
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-18.04]
        java: [8, 11, 14, 17]
      fail-fast: false   
    name: Test common-utils with JDK ${{ matrix.java }}, ${{ matrix.os }}
```

Reply:
Ignore me :) 
Just saw the description and it makes sense.